notebooks/5001_process-CEDS.py

```diff
@@ -290,6 +301,35 @@ def update_units_r(em, units):
     check_like=True,
 )
 
+# %%
+ceds_esgf = CEDS_CMIP_PROCESSED_DB.load()
+# ceds_esgf
+
+# %%
+ur = pint.get_application_registry()
+
+# %%
+species_renamings = {
+    "NMVOC": "VOC",
+    "SO2": "Sulfur",
+}
+for (species, units), sdf_esgf in ceds_esgf.groupby(["species", "unit"]):
```

Comment:
Here is where we check that what we have processed matches what you get from ESGF exactly